submissions/Valkirin/js-dom/script.js

```diff
@@ -0,0 +1,103 @@
+      'Genuine feelings for mutually beneficial solutions or agreements in your relationships. Value and respect people by understanding a "win" for all is ultimately a better long-term resolution than if only one person in the situation had gotten their way. Think Win-Win isn\'t about being nice, nor is it a quick-fix technique. It is a character-based code for human interaction and collaboration.'
+  },
+  {
+    name: "First - hear, then be heared",
+    img: "./img/hear.jpg",
+    description:
+      "Use empathetic listening to genuinely understand a person, which compels them to reciprocate the listening and take an open mind to being influenced by you. This creates an atmosphere of caring, and positive problem solving."
+  },
+  {
+    name: "Synergy",
+    img: "./img/synergy.jpg",
+    description:
+      "Combine the strengths of people through positive teamwork, so as to achieve goals that no one could have done alone."
+  },
+  {
+    name: "Sharpen saw",
+    img: "./img/saw.jpg",
+    description:
+      "Balance and renew your resources, energy, and health to create a sustainable, long-term, effective lifestyle. It primarily emphasizes exercise for physical renewal, good prayer (meditation, yoga, etc.) and good reading for mental renewal. It also mentions service to society for spiritual renewal."
+  }
+];
+
+const container = document.createElement("div");
+container.classList.add("container");
+document.body.appendChild(container);
+
+const content = document.createElement("div");
+content.classList.add("content");
+container.appendChild(content);
+
+const contentTitle = document.createElement("h2");
+contentTitle.classList.add("content_title");
+contentTitle.textContent = habbits[0].name;
+content.appendChild(contentTitle);
+
+const descriptionPar = document.createElement("p");
+descriptionPar.classList.add("description_text");
+descriptionPar.textContent = habbits[0].description;
+content.appendChild(descriptionPar);
+
+const menu = document.createElement("div");
+container.appendChild(menu);
+
+const navigation = document.createElement("ul");
+navigation.classList.add("navigation");
+menu.appendChild(navigation);
+
+habbits.forEach((val, key) => {
+  const createListName = document.createElement("li");
+  createListName.classList.add("listel");
+  createListName.textContent = habbits[key].name;
+  navigation.appendChild(createListName);
+});
+
+const containerImg = document.createElement("div");
+containerImg.classList.add("content_img");
+container.appendChild(containerImg);
+
+const contentImg = document.createElement("img");
+contentImg.classList.add("image");
+contentImg.setAttribute("src", habbits[0].img);
+containerImg.appendChild(contentImg);
+
+const changeMenu = document.querySelector("ul");
+
+changeMenu.addEventListener("click", evt => {
+  const getList = document.querySelectorAll("li");
+  getList.forEach(element => {
+    element.classList.remove("clicked");
+  });
+
+  habbits.forEach((val, key) => {
```

Comment:
The combination of `forEach` and `if` statement can be replaced by array method `find`. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
Combination replaced